README.md

```diff
@@ -47,60 +53,25 @@ All templates from this repository come as part of the [Amido.Stacks.CQRS.Events
 
 #### Template installation
 
-To install the templates locally you'll need to download the [Amido.Stacks.CQRS.Events.Templates](https://www.nuget.org/packages/Amido.Stacks.CQRS.Templates/) NuGet package.
-
-Then you can install it locally to your machine via the command line (the version provided is just an example, please consult the NuGet page for an up-to date version)
+For the latest template version, please consult the Nuget page [Amido.Stacks.CQRS.Events.Templates](https://www.nuget.org/packages/Amido.Stacks.CQRS.Templates/). To install the templates to your machine via the command line:
 
 ```shell
-dotnet new --install Amido.Stacks.CQRS.Templates::0.0.85g
+dotnet new --install Amido.Stacks.CQRS.Templates::0.0.101
```

Comment:
This should be `Amido.Stacks.CQRS.Events.Templates`

Reply:
Corrected nuget package link.